contracts/enclaves/MarketV1.sol

```diff
@@ -159,6 +159,7 @@ contract MarketV1 is
     uint256 public jobIndex;
 
     IERC20 public token;
+    uint256 public constant extraDecimals = 12;
```

Comment:
Make the variable Capital case so that it is consistent with rest of the constants in other contracts

Reply:
fixed